src/Execute/quickpick.ts

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// TODO: Check the license and add license
+// This draft is mostly referenced from `vscode-extension-samples/quickinput-sameple`
+// https://github.com/microsoft/vscode-extension-samples/tree/main/quickinput-sample
+
+import * as vscode from 'vscode';
+import {globalBackendMap} from '../Backend/Backend';
+import { Backend } from '../Backend/API';
+import {Balloon} from '../Utils/Balloon';
+
+
+interface inferenceCommand {
+  backend: Backend;
+  modelPath: vscode.Uri;
+  step: number;
+  totalSteps: number;
+}
+
+// NOTE: This is temporal function to show its successful implementation
+// TODO: Remove this after execution has landed.
+function print_command(cmd: Partial<inferenceCommand>) {
+  let _backend = cmd.backend;
+  let _modelPath = cmd.modelPath;
+
+  if(_backend == undefined || _modelPath == undefined) {
+    return;
+  }
+  
+  let executor = _backend.executor();
+  if(executor == undefined) {
+    return;
+  }
+
+  let command = executor.inference(_modelPath.path, ["--input-spec", "any"]);
+  Balloon.info(command.str())
+}
+
+export async function selectBackendExecutor(context: vscode.ExtensionContext) {
+  const cmd = {} as Partial<inferenceCommand>;
+
+  const quickPick = vscode.window.createQuickPick();
+  quickPick.items = Object.keys(globalBackendMap).map(label => ({ label }));
+  quickPick.onDidChangeSelection(selection => {
+    if(selection[0]) {
+      cmd.backend = globalBackendMap[selection[0].label];
+      console.log(cmd);
+      quickPick.dispose();
+      selectModel(context, cmd).then(c => {
+        if(c == undefined) {
+          return;
+        }
+        console.log(c);
+        print_command(c);
+      });
+    }
+  });
+  quickPick.onDidHide(() => quickPick.dispose());
+  quickPick.show();
+}
+
+// This is mostly copied from getImportCfgFilepath in `Utils/Helpers.ts`
+// TODO: Change to better implementation
+const selectModel = async (context: vscode.ExtensionContext, cmd: Partial<inferenceCommand>) => {
+  const options: vscode.OpenDialogOptions = {
+    canSelectMany: false,
+    openLabel: 'Select Model to Infer',
+    filters: {
+      /* eslint-disable-next-line @typescript-eslint/naming-convention */
+      '.tvn Files': ['tvn']
```

Comment:
Q. Is this `quickpick` for just `tvn`? or also `tflite`?

Reply:
for just tvn. When I first designed this draft, I haven't considered about `executorTflite`. I'll add todo. Thank you!